libsolidity/analysis/ConstantEvaluator.cpp

```diff
@@ -49,7 +49,7 @@ bool fitsPrecisionExp(bigint const& _base, bigint const& _exp)
 
 	size_t const bitsMax = 4096;
 
-	unsigned mostSignificantBaseBit = boost::multiprecision::msb(_base);
+	auto mostSignificantBaseBit = boost::multiprecision::msb(_base);
```

Comment:
We generally prefer explicit types:
```suggestion
	unsigned long mostSignificantBaseBit = boost::multiprecision::msb(_base);
```